web/frontend/src/components/remote-control-cards.vue

```diff
@@ -563,6 +567,8 @@ const createAppConnectionManager = () => {
 
   const stop = () => {
     window.clearTimeout(timeout);
+    statusStream?.cancel();
+    statusStream = null;
```

Comment:
@edaniels I added these but let me know if I should pull them out. Feels like if we are stopping we should cancel the stream, but want to double check.